content/en/docs/zero-code/java/quarkus.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Quarkus instrumentation
+linkTitle: Quarkus
+---
+
+## Quarkus OpenTelemetry instrumentation
+
+[Quarkus](https://quarkus.io/) is an open source framework designed to help
+software developers build efficient cloud native applications both with JVM and
+Quarkus native image applications.
+
+Quarkus uses extensions to provide optimized support for a wide range of
+libraries. The
+[Quarkus OpenTelemetry extension](https://quarkus.io/guides/opentelemetry)
+provides:
+
+- Out of the box instrumentation
+- OpenTelemetry SDK autoconfiguration, supporting almost all system properties
+  defined for the [OpenTelemetry SDK](/docs/languages/java/configuration/)
+- [Vert.x](https://vertx.io/) based OTLP exporter
+- The same instrumentations can be used with native image applications, which
+  are not supported by the OpenTelemetry Java agent.
+
+> [!NOTE] Quarkus OpenTelemetry instrumentation is maintained and supported by
+> Quarkus. For details, please check the
+> [Quarkus community support page](https://quarkus.io/support/).
+
+Quarkus can also be instrumented with the [OpenTelemetry Java agent](../agent/)
+if you are not running a native image application.
+
+## Getting started
+
+To enable OpenTelemetry in your Quarkus application, add the
+`quarkus-opentelemetry` extension dependency to your project.
+
+{{< tabpane >}} {{< tab header="Maven (`pom.xml`)" lang=Maven >}}
+
+```xml
+<dependency>
+    <groupId>io.quarkus</groupId>
+    <artifactId>quarkus-opentelemetry</artifactId>
+</dependency>
+```
+
+{{% /tab %}} {{% tab header="Gradle (`build.gradle`)" lang=Gradle %}}
+
+```kotlin
+implementation("io.quarkus:quarkus-opentelemetry")
+```
+
+{{% /tab %}} {{< /tabpane>}}
+
+Only the **tracing** signal will be enabled by default. To enable **metrics**
+and **logs**, you need to add the following configuration to your
+`application.properties` file:
+
+```properties
+quarkus.otel.metrics.enabled=true
+quarkus.otel.logs.enabled=true
+```
+
+OpenTelemetry logging is available after Quarkus 3.16.0.
+
+The remaining configurations are available in the
+[Quarkus OpenTelemetry configuration reference](https://quarkus.io/guides/opentelemetry#configuration-reference).
```

Comment:
```suggestion
For details concerning these and other configuration options, see
[OpenTelemetry Configuration Reference](https://quarkus.io/guides/opentelemetry#configuration-reference).
```

Reply:
We should keep `[Quarkus OpenTelemetry configuration reference]` because that page belongs to Quarkus and can cause ownership confusion.

Reply:
There's no confusion, it links to Quarkus.io.